code-challenges.js

```diff
@@ -12,53 +12,86 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
-
-
-
+const divideByThree = arg =>{
+    if(arg%3 === 0){
+        console.log(`${arg} is divisible by three`);        
+    } else if (arg%3 !== 0){
+        console.log(`${arg} is not divisible by three`);
+        
+    }
+}
+
+divideByThree(num1);
+divideByThree(num2);
+divideByThree(num3);
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitalize = arg => {
+    let newArray = arg.map(word =>
+        word.replace(/^\w/, c => c.toUpperCase()))   
+    return newArray;
+    }
```

Comment:
cool use of regex here. but if you are going to use it, make sure you add some comments on what the code is actually doing. i say this because if this was an interview and you wrote out regex, your interviewer 9 1/2  times out of 10 will ask you what exactly is happening.